HOW-TO-LINT-YOUR-CHANGE.md

```diff
@@ -0,0 +1,58 @@
+# How to lint your change
+
+This document describe how to setup automatically or manually linting your change.
+
+## Prerequisites
+- `gofumpt` should be installed. `go install mvdan.cc/gofumpt@latest`
+  - go version should be equal or later than v1.18.0
+- `goimports` should be installed. eg. `go install golang.org/x/tools/cmd/goimports@latest`
+
+## Setup Git Hook
+This will apply code formating automatically when you commit to git repository. So you can pass the linting tests registered in Klaytn Circle CI.
```

Comment:
```suggestion
This will apply code formatting automatically when you commit your change. So you can pass the lint tests registered in Klaytn's CI.
```